app/javascript/src/question_autocomplete.js

```diff
@@ -13,26 +13,30 @@
  *
  **/
 
-
-const utilities = require('./utilities');
+const utilities = require("./utilities");
 const mergeObjects = utilities.mergeObjects;
-const Question = require('./question');
+const Question = require("./question");
 
 const SELECTOR_HINT = ".govuk-hint";
-const SELECTOR_LABEL = "label h1, label h2, legend h1, legend h2";
```

Comment:
Why did we remove legend here? Because this component type doesn't have them?

Reply:
yep, the autocomplete is a single field component, so it only has a label not a legend